src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/ciagkpi/PefCiagKpiService.kt

```diff
@@ -45,6 +45,14 @@ class PefCiagKpiService(
    * - If the prisoner already has had their Induction created. Then this person is likely to
    * be a coming back into prison and needs to resume their Reviews.
    */
+  /* TODO - the logic is wrong, and needs to be something like:
```

Comment:
This is a worry - have you created a ticket to update it?

Reply:
Not yet, but will do

Reply:
RR-1198